src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CoreCaseDataApi2.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.divorce.systemupdate.service;
+
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@FeignClient(
+    name = "core-case-data-api2",
+    primary = false,
+    url = "${core_case_data.api.url}",
+    configuration = CoreCaseDataConfiguration.class
+)
+public interface CoreCaseDataApi2 {
```

Comment:
Maybe we could have a comment explaining when to use this instead of CoreCaseDataApi, or adjust the name to describe the extra functionality it provides?

Reply:
my imagination fails me, I want it to be something that makes it obvious it's like corecasedataapi but anything saying it's just to get the value we need out too long 
eg CoreCaseDataApiWithStateModifiedDate 
Happy to go with that if you think it's an improvement on CoreCaseDataApi2 tho

Reply:
name changed